XenAdmin/Wizards/GenericPages/SelectVMStorageWithMultipleVirtualDisksPage.cs

```diff
@@ -290,23 +290,26 @@ private void PopulateSrComboBox(List<ToStringWrapper<SR>> commonSRs)
 			foreach (var toStringWrapper in commonSRs)
 			{
 				ulong requiredSpace = 0;
+			    bool isSuitableForAll = true;
 				foreach (var kvp in m_vmMappings)
 				{
 					foreach (IStorageResource resourceData in ResourceData(kvp.Key))
 					{
-						if (IsExtraSpaceNeeded(resourceData.SR, new XenRef<SR>(toStringWrapper.item)))
+						if (IsExtraSpaceNeeded(resourceData.SR, toStringWrapper.item))
 						{
 							requiredSpace += resourceData.RequiredDiskCapacity;
+						    if (!SrIsSuitable(resourceData.SR) || !SrIsSuitable(toStringWrapper.item))
+						        isSuitableForAll=false;

```

Comment:
I don't think we ever set the home server during a migration (see 1119, where we renamed "home server" to "target server" in this wizard to avoid confusion)